pkgs/tools/X11/oblogout/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, intltool, file, pythonPackages, cairo }:
+
+pythonPackages.buildPythonApplication rec {
+  name = "oblogout-${version}";
+  version = "0.2";
```

Comment:
0.3 is the development version and it has not been released.

I am going to use a fork repository for the source code, with more recent commits. The same is used in the package for Manjaro Linux. Some patches will not be needed anymore.